recipes/hackgap/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hackgap" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 35c5705d8ca1de0d8064c9869730b65f596a0a378a9d14f8a58368e4a71aab22
+
+build:
+  number: 0
+  entry_points:
+    - hackgap = hackgap.hackgap.hackgap_main:main
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.12
+  run:
+    - python >=3.12
+    - numpy >=2
+    - numba >=0.60
+    - pytest
```

Comment:
Is it needed to run the app ? On only to test it ?

Reply:
pytest is not needed to run the app. Should I remove it?

Reply:
yes, please!
Same for any other not needed dependency if there are more.

Reply:
Done! The other dependencies are necessary to run the tools.